ckanext/datastore/plugin.py

```diff
@@ -282,21 +282,9 @@ def before_show(self, resource_dict):
                 controller='ckanext.datastore.controller:DatastoreController',
                 action='dump', resource_id=resource_dict['id'])
 
-        connection = None
+        if 'datastore_active' not in resource_dict:
+            resource_dict[u'datastore_active'] = False
```

Comment:
this part makes me sad because it kills the speedup we get from serving the validated data dict straight from solr